src/docker/shared.rs

```diff
@@ -714,13 +714,24 @@ pub(crate) fn docker_mount(
     paths: &DockerPaths,
     mount_cb: impl Fn(&mut Command, &Path, &Path) -> Result<()>,
     mut store_cb: impl FnMut((String, String)),
+    msg_info: &mut MessageInfo,
 ) -> Result<()> {
+    let mut warned = false;
```

Comment:
Avoid multiple warnings, since a single warning should be sufficient to notify the user that their syntax is invalid.